src/Model/StoModel.cpp

```diff
@@ -593,6 +985,71 @@ void StoModel::copyCoreObjective(double * obj, int stg)
 	}
 }
 
+void StoModel::copyCoreQuadrativeObjective(CoinPackedMatrix * &qobj_coupling, CoinPackedMatrix * &qobj_ncoupling, int stg)
```

Comment:
`...Quadrative...` in function name?

Reply:
Fixed (https://github.com/Argonne-National-Laboratory/DSP/pull/143/commits/56b28f5b94534977fed61f1049c1d0b7501e691a)